pkg/authorization/static/static.go

```diff
@@ -60,17 +60,34 @@ func (saConfig StaticAuthorizationConfig) Matches(a authorizer.Attributes) bool
 	isAllowed := func(staticConf string, requestVal string) bool {
 		if staticConf == "" {
 			return true
-		} else {
-			return staticConf == requestVal
 		}
+		return staticConf == requestVal
+	}
+	isGroupAllowed := func(configGroups, requestGroups []string) bool {
+		if len(configGroups) == 0 {
+			return true
+		}
+		configGroupSet := make(map[string]struct{})
+		for _, group := range configGroups {
+			configGroupSet[group] = struct{}{}
+		}
+		for _, group := range requestGroups {
+			if _, exists := configGroupSet[group]; exists {
+				return true
+			}
+		}
+		return false
 	}
 
 	userName := ""
+	userGroups := []string{}
 	if a.GetUser() != nil {
 		userName = a.GetUser().GetName()
+		userGroups = a.GetUser().GetGroups()
 	}
 
-	if isAllowed(saConfig.User.Name, userName) &&
+	if (saConfig.User.Name == "" || isAllowed(saConfig.User.Name, userName)) &&
```

Comment:
I think the previous solution was sufficient. If there is no config (empty string) for `user` or any other resourceAttribute, it defaults to true.

The unit tests show those permutations of user / no user, group / no group, allowed.